### PR TITLE
data:StarKnightt

### PR DESCRIPTION
**Pull request by StarKnightt**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5874"></a>

### CODE REVIEW

File: data/StarKnightt.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/starknight__/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/prasenjitnayak/",
+      "icon": "FaLinkedIn"
```
Comment (ChinmayMhatre) [inline data/StarKnightt.json:19]:
```suggestion
      "icon": "FaLinkedin"
```

File: data/StarKnightt.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/StarKnightt",
+      "icon": "FaGitHub"
```
Comment (ChinmayMhatre) [inline data/StarKnightt.json:24]:
```suggestion
      "icon": "FaGithub"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks awesome! A few changes need to be done for which I have left comments

**Comment by ChinmayMhatre** [comment]
Please check the rest of the icons from the set of available icons at : https://linkfree.io/icons

File: data/StarKnightt.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/prasenjitnayak/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/StarKnightt",
+      "icon": "FaGitHub"
```
Comment (khushalarora) [inline data/StarKnightt.json:24]: You only need to replace this with **' FaGithub '**, &nbsp;as suggested by @ChinmayMhatre.
Rest everything looks good :+1: 
```Suggestion
      "icon": "FaGithub"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @StarKnightt, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by StarKnightt** [comment]
> looks good

Thank YOU

**Pull request merged**